UnitTests/BraintreePayPalTests/BTPayPalVaultRequest_Tests.swift

```diff
@@ -123,7 +123,13 @@ class BTPayPalVaultRequest_Tests: XCTestCase {
             startDate: "test-date",
             pricing: billingPricing
         )
-        
+
+        let billingAmountBreakdown = BTRecurringBillingAmountBreakdown(
```

Comment:
These test updates should be moved to the `BTPayPalCheckoutRequest` tests instead